build.sbt

```diff
@@ -47,4 +61,14 @@ scalacOptions in (Compile, doc) ++= Seq(
 
 scalacOptions in (Test, doc) ++= Seq("-groups", "-implicits")
 
+// This fixes a class loader problem with scala.Tuple2 class, scala-2.11, Spark 2.x
+// "scala.ScalaReflectionException: class scala.Tuple2 in JavaMirror with sbt.classpath.ClasspathFilter@61de7710 of type class sbt.classpath.ClasspathFilter with classpath [<unknown>] and parent being sbt.classpath.ClasspathUtilities$$anon$1@4b18bc2d of type class sbt.classpath.ClasspathUtilities$$anon$1 with classpath"
+fork in Test :=
```

Comment:
It was only failing with Scala-2.11, so this was to minimize the change in earlier versions